dvc/scm/base.py

```diff
@@ -92,8 +92,7 @@ def ignore_list(self, p_list):
         return [self.ignore(path) for path in p_list]
 
     def add(self, paths):
-        """Makes SCM start tracking every path from a specified list of paths.
-        """
+        """Makes SCM track every path from a specified list of paths."""
```

Comment:
I shortened this to be within 80 chars. There's some issue with black on how it handles docstrings.